src/NumericalAlgorithms/SphericalHarmonics/IO/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+spectre_target_sources(
```

Comment:
Will do this! A general question: can you not have subdirectories that are part of a library defined in the parent folder? i.e. can you not make a subfolder without also making it its own library?

Reply:
In general yes you can have subdirectories that are part of the library of the parent folder. That's what you had originally. However, we don't want that in this case because we don't want the `SphericalHarmonics` library to depend on the `IO` library. That's the main reason to make the subdirectory is because we actually wanted a new library.